src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -31,23 +32,10 @@
     private PasswordField passInput;                // Takes input for username and password
     private ClientContext clientContext;            //
 
-    // main page vars
-    private HBox hboxClient;
-    private HBox hboxInput;
-    private VBox userVBox;
-    private VBox chatVBox;
-    private VBox convosVBox;
-    private BorderPane container;
-    private Button sendButton;
-    private Button updateButton;
-    private Button addConvoButton;
-    private Text userTitle;
-    private Text chatTitle;
-    private Text convosTitle;
-    private TextFlow userTf;
-    private TextFlow chatTf;
-    private TextFlow convosTf;
-    private TextField input;
+    // list of conversations
+    private ObservableList<String> convoList;      // list of conversations
```

Comment:
nit: I'd stick with one style of comments, either above or to the side. Generally I'd say above is better because of column widths, and javadoc style comments are even better.